### PR TITLE
MBM-280 Telekinetic offensive tier one

### PR DESCRIPTION
**Pull request by MattKirchh**

### CODE REVIEW

File: src/main/java/com/mineboundteam/minebound/item/tool/MyrialMachete.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        }
+    }
+
+    @Override
+    public boolean onDroppedByPlayer(ItemStack item, Player player) {
+        player.getInventory().removeItem(item);
+        return false;
+    }
+
+    // TODO: machete can be placed in storage containers like chests, need to prevent that somehow
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/item/tool/MyrialMachete.java:45]: Maybe look into 
```
   public boolean canFitInsideContainerItems() {
      return true;
   }
```

Comment (MattKirchh) [reply src/main/java/com/mineboundteam/minebound/item/tool/MyrialMachete.java:45]: Despite what the name implies, the function is only used for shulker boxes and has no effect on traditional storage containers.

Comment (insertStringHere) [reply src/main/java/com/mineboundteam/minebound/item/tool/MyrialMachete.java:45]: Could you use inventoryTick to check if the player has a container open and cause it to vanish as soon as there is?

Comment (MattKirchh) [reply src/main/java/com/mineboundteam/minebound/item/tool/MyrialMachete.java]: Yup, that does work. I don't think it's a perfect solution but it works well enough. Pushed.

Comment (insertStringHere) [reply src/main/java/com/mineboundteam/minebound/item/tool/MyrialMachete.java]: it's not perfect but I think the alternative would be trying to figure out how to make a custom itemstack or something similar

**Review by solidwaterslayer — APPROVED** [review]

**Review by insertStringHere — APPROVED** [review]
Still allows the player to cast while holding the sword, but that can be a bugfix

**Review by insertStringHere — APPROVED** [review]

**Pull request merged**